dask-gateway/dask_gateway/client.py

```diff
@@ -29,6 +30,17 @@
 
 del comm
 
+# Configure default AsyncClient backend
+HTTP_CLIENTS = {
+    "curl": "tornado.curl_httpclient.CurlAsyncHTTPClient",
+    "simple": "tornado.simple_httpclient.SimpleAsyncHTTPClient",
+}
+
+HAS_PYCURL = False
+if importlib.util.find_spec("pycurl") is not None:
+    HAS_PYCURL = True
```

Comment:
I'd avoid doing this at import time, and instead try to import when setting up the client (and erroring appropriately if not installed). This allows users to install after starting python if they forgot to, which is something I often do for other packages.